autopush_rs/src/server/mod.rs

```diff
@@ -223,6 +223,10 @@ pub extern "C" fn autopush_server_new(
                 .expect("poll interval cannot be 0"),
         };
         opts.message_table_names.sort_unstable();
+        opts.current_message_month = opts.message_table_names
```

Comment:
a number of the db methods do this same call to get current_message_month. we can improve that later though since they're getting refactored

Reply:
Yes, and they do weird things since its not 'for-sure' that there's a last() on the vec, while now its guaranteed there'll be a current month.